src/test/kotlin/bloombox/client/shop/ShopInfoTest.kt

```diff
@@ -112,73 +116,81 @@ class ShopInfoTest: ClientRPCTest() {
    */
   @test
   fun testZipcheckKnownGood() {
-    val responseOne = client.platform.shop().checkZipcode("95120", ShopClient.ShopContext(
-          partner = partnerID,
-          location = locationID))
-
-    val responseTwo = client.platform.shop().checkZipcode("94404", ShopClient.ShopContext(
-          partner = partnerID,
-          location = locationID))
-
-    assertNotNull(responseOne, "response from server for zipcheck 1 should not be null")
-    assertNotNull(responseTwo, "response from server for zipcheck 2 should not be null")
-    assertTrue(responseOne.supported, "response from server for zipcheck 1 should grant access")
-    assertTrue(responseTwo.supported, "response from server for zipcheck 2 should grant access")
-    assertEquals(responseOne.deliveryMinimum, (50.0).toFloat(), "response should specify correct delivery minimum for zipcheck 1")
-    assertEquals(responseTwo.deliveryMinimum, (200.0).toFloat(), "response should specify correct delivery minimum for zipcheck 2")
+    withClient({ client ->
+      val responseOne = client.platform.shop().checkZipcode("95120", ShopClient.ShopContext(
```

Comment:
Are these Zip codes hard coded into the test to match against preset zip codes in the code base, or are we testing against Caliva's data specifically for now?

Reply:
ie. what is a "known good zip code" in relation to these tests? A test club? Caliva's real data? etc.

Reply:
we talked about this offline, but updating here for posterity's sake. yes, these are hard coded to work with the test partner/location pair and API key.